pyart/aux_io/sinarame_h5.py

```diff
@@ -0,0 +1,319 @@
+"""
+pyart.aux_io.SINARAME_h5
+====================
```

Comment:
The docstring name should match the name of the file in case.  Also the underline needs to extend to the end of the line above.